src/rabbitmq.js

```diff
@@ -293,16 +290,13 @@ class RabbitMQ {
    */
   publishEvent (exchange: string, content: Object): Bluebird$Promise<void> {
     return Promise.try(() => {
-      const bufferContent = this._validatePublish(exchange, content, 'events')
-      const jobMeta = {
-        appId: this.name,
-        timestamp: Date.now()
-      }
-      this.log.info({ event: exchange, job: content, jobMeta: jobMeta }, 'Publishing event')
+      this._validatePublish(exchange, content, 'events')
```

Comment:
nit: seems like you can break this and above changes to a helper function as well. also I would not write unit test for helper functions, just keep the current ones passing and coverage will tell us if we need to write another test